src/DynamoCore/DynamoCore.csproj

```diff
@@ -172,9 +174,9 @@
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.xml" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.Migrations.xml" DestinationFolder="$(OutputPath)" />
     <!--Enable these if/when we need to keep old binaries for patch installer compatability-->
-    <Copy SourceFiles="@(LegacyBinariesToBinFolder)" DestinationFolder="$(OutputPath)" />
-    <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />
-     <Copy SourceFiles="@(LegacyBinariesToBinResourcesFolder)" DestinationFolder="$(OutputPath)\en-US\" />
+    <Copy SourceFiles="@(LegacyBinariesToBinFolder)" DestinationFolder="$(OutputPath)" Condition=" '$(TargetFramework)' == 'net48' "/>
+    <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" Condition=" '$(TargetFramework)' == 'net48' "/>
+     <Copy SourceFiles="@(LegacyBinariesToBinResourcesFolder)" DestinationFolder="$(OutputPath)\en-US\" Condition=" '$(TargetFramework)' == 'net48' "/>
```

Comment:
Unrelated to the migration of workspace-dep view extension but I realized that there shouldn't be a need to copy these legacy files for .net6 builds.

Reply:
I don't think we need to copy these files for .net6 builds. The first version of Revit/Civil using net[6,8] will be full installer builds. That is also true for all Pre-releases up to RTM.